pcf/particle/aws/glacier/glacier_vault.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from botocore.exceptions import ClientError
+from pcf.core.aws_resource import AWSResource
+from pcf.core import State
+from pcf.util import pcf_util
+
+logger = logging.getLogger(__name__)
+
+class GlacierVault(AWSResource):
+    """
+    This is the implementation of Amazon's Glacier service. Additional functions that are callable on this particle
+    are upload_archive, initiate_job, list_jobs, delete_archive, delete_job, add_tags_to_vault, list_tags_for_vault
+    """
+    flavor = "glacier"
+
+    state_lookup = {
+        "missing": State.terminated,
+        "active": State.running,
+        "inactive": State.terminated
+    }
+
+    equivalent_states = {
+        State.running: 1,
+        State.stopped: 0,
+        State.terminated: 0
+    }
+
+    START_PARAMS_FILTER = {
```

Comment:
dont need filter since everything in def will get passed in to create_vault.

Reply:
@anovis Shouldn't we keep this as is since we also have custom_configs in the desired_state_definition and that does not get passed in the initial creation of the vault. The only  accepted arguments for create_vault() are vaultName and accountId(which I will address as a default variable if none is specified per your request).